build/Jenkinsfile.branch

```diff
@@ -124,7 +124,7 @@ pipeline {
           withCredentials([usernamePassword(credentialsId: 'matterbuild-docker-hub', usernameVariable: 'DOCKER_USER', passwordVariable: 'DOCKER_PASS')]) {
             sh 'docker login --username ${DOCKER_USER} --password ${DOCKER_PASS}'
             sh """
-            export TAG='${env.BRANCH_NAME}.${env.BUILD_NUMBER}.${SERVER_GIT_COMMIT}'
```

Comment:
Does this mean we will overwrite previous images from the same branch? I don't think that's an issue, but I wanted to check that it won't affect upgrades or anything.

Reply:
yes it will override and that is ok